gcn_classic_to_json/utils.py

```diff
@@ -0,0 +1,11 @@
+from astropy.time import Time
+
+# Zero point for Truncated Julian Day according to
+# https://en.wikipedia.org/wiki/Julian_day.
+TJD0 = (2440000, 0.5)
+
+
+# Date must be in Truncated Julian Date format
+# Time must be in Seconds of Day format
```

Comment:
Transform these comments into a Numpy-style doctstring for the function. See https://numpydoc.readthedocs.io/en/latest/format.html.